### PR TITLE
WIP: Enable openj9 jdk8 mac code signing

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Signed-off-by: Andrew Leonard <andrew_m_leonard@uk.ibm.com>

### CODE REVIEW

File: sign.sh [hunk 1]
```diff
@@ -150,7 +111,7 @@ function extractArchive {
   fi
 }
 
-if [ "${OPERATING_SYSTEM}" != "windows" ] && [ "${OPERATING_SYSTEM}" != "mac" ]; then
```
Comment (gdams) [inline sign.sh:153]: This is still needed for hotspot

File: build-farm/platform-specific-configurations/mac.sh [hunk 2]
```diff
@@ -41,6 +41,9 @@ then
     export COMPILER_WARNINGS_FATAL=false
     echo "Compiler Warnings set to: $COMPILER_WARNINGS_FATAL"
   fi
+  if [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]; then
+    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-toolchain-type=clang"
```
Comment (gdams) [inline build-farm/platform-specific-configurations/mac.sh:45]: this isn't openj9 specific. Applies to both hotspot and j9

**Comment by karianna** [comment]
@andrew-m-leonard - merge conflicts due to us having to revert some earlier work.

**Comment by andrew-m-leonard** [comment]
This PR should cover these changes: https://github.com/AdoptOpenJDK/openjdk-build/pull/1688

**Pull request closed without merging**